privacyidea/lib/config.py

```diff
@@ -716,24 +715,13 @@ def get_token_list():
     module_list.add("privacyidea.lib.tokens.pushtoken")
     module_list.add("privacyidea.lib.tokens.indexedsecrettoken")
     module_list.add("privacyidea.lib.tokens.webauthntoken")
-    #module_list.add(".tokens.tagespassworttoken")
-    #module_list.add(".tokens.vascotoken")
 
-    # Dynamic Resolver modules
-    # TODO: Migration
-    # config_modules = config.get("privacyideaResolverModules", '')
-    config_modules = None
-    log.debug("{0!s}".format(config_modules))
-    if config_modules:
-        # in the config *.ini files we have some line continuation slashes,
-        # which will result in ugly module names, but as they are followed by
-        # \n they could be separated as single entries by the following two
-        # lines
-        lines = config_modules.splitlines()
-        coco = ",".join(lines)
-        for module in coco.split(','):
-            if module.strip() != '\\':
-                module_list.add(module.strip())
+    # Dynamic token modules
+    dynamic_token_modules = get_app_config_value("PI_TOKEN_MODULES")
+    if dynamic_token_modules:
+        # In the pi.cfg you can specify a list or set of 3rd party token modules like
+        #    PI_TOKEN_MODULES = [ "myproj.tokens.tok1", "myproj.tokens.tok2" ]
+        module_list.update(dynamic_token_modules)
```

Comment:
I know it documented several times but people tend not to read documentation (at least i ran into this...).
We could use
`module_list.update(dynamic_token_modules if isinstance(dynamic_token_modules, list) else [dynamic_token_modules])`
to allow a single string in the configuration. What do You think?

Reply:
Hm, the documentation seems not quite correct.
``module_list`` is actually a **set**. And the ``update`` method of sets works for lists/sets and strings.

    module_list.update("mytoken")

and 
    
    module_list.update(["mytoken"])

work the same.

Reply:
actually not, in the first case every character of the string will be added to the set

Reply:
:-( I should have user a string longer than one character.

Again. We have a similar issue with other configurations like the ``SUPERUSER_REALM`` which also expects a list and would end up in wired results, if you would give a string.
So I am hesitant to do a failsafe here and not have a failsafe somewhere else, which is used much more often.

Reply:
I will do a helper function, which we might use at other places in the future.

Reply:
done in d1f5619c